app/prefix.go

```diff
@@ -40,7 +39,7 @@ func SetConfig(testnet bool) {
 
 	config := sdk.GetConfig()
 	config.SetCoinType(uint32(CoinType))
-	config.SetFullFundraiserPath(fmt.Sprintf("m/44'/%d'/0'/0/0", CoinType))
+	config.SetPurpose(Purpose)
```

Comment:
is there a way to verify the Config with a test?

Reply:
Good point, looking into this now.